EBA/Taschenrechner/Informatikrechner/Zahlensystem.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Taschenrechner
+{
+    public partial class fm_Zahlensystem : BaseForm
+    {
+        string eingabe = string.Empty;
+        public fm_Zahlensystem()
+        {
+            InitializeComponent();
+            eingabe = Eingabe();
+        }
+
+        public  string Eingabe()
+        {
+            ShowMessage("Bitte gib die zu konvertierende Zahl ein");
+            var eingabe = helper.ToString();
+            int[] digits = eingabe.ToCharArray().Select(x => (int)Char.GetNumericValue(x)).ToArray();
+            var pos = digits.Length;
+            pos--; //zur Vermeidung von Nullpointern
+
+            //hier wird überprüft welche Zahlen in der Eingabe vorhanden sind und daraus ermittelt welche Zahlensysteme möglich sind
+            //deaktiviert RadioButtons für ungültige Zahlensysteme
+            while (pos > 0)
```

Comment:
you can't check the number system with the number. Because the number:

10 can be:
decimal
binary
octal
Ternär

only numbers with a `9` like `89` can be trusted that they are decimal.
A number with all numbers less then 8 can be octal and decimal.
And so on.

Reply:
i improved this shit :)

wanna look over it @paule96 ? :)